.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: cpp
 
+install:
+ - curl -O https://www.libsdl.org/release/SDL2-2.0.8.tar.gz
```

Comment:
Wir sollten das meiner Meinung nach noch irgendwo dokumentieren (zumindest in der README?) welche libs wir nutzen.

Reply:
Siehe ThirdParty Ordner, in der sind alles Dependencies drin. Können wir gerne aber nochmal in der Readme festhalten